release_notes/3056-release-notes.adoc

```diff
@@ -0,0 +1,124 @@
+[id="3056-release-notes"]
+= Red Hat Advanced Cluster Security for Kubernetes version 3.0.56
+include::modules/common-attributes.adoc[]
+:context: 3056-release-notes
+
+toc::[]
+
+[role="_abstract"]
+{product-title} version 3.0.56 includes feature enhancements, bug fixes, scale improvements, and other changes.
+
+*Release date:* February 24, 2021
+
+== New features
```

Comment:
I think we're supposed to add explicit anchors for all of these headings, both in modules and assemblies